### DISCUSSION TITLE
MPV video

### DISCUSSION THREAD

**Discussion by DzmitryMurzin** [Joshua-Riek/ubuntu-rockchip#567] [Q&A]
There's a comment in the Highlights - "MPV video player capable of smooth 4k video playback".

But I have only green screen for any 4K videos (Orange pi 5). Is it an issue or I'm missing something?

**Comment by Joshua-Riek** [comment]
You are likely trying to play HDR content, it does not work with the Wayland desktop environment.

**Comment by glass-penguin** [comment]
The mpv starts playing normally, but after a random amount of time, gnome crashes.
Celluloid and smplayer behave the same way.
